src/css/footer.css

```diff
@@ -151,9 +152,9 @@ section#links-footer {
 .operational ul li h1 {
```

Comment:
Ik denk dat deze selector niet goed is. Want op een pagina mag een `<h1>` maar 1 keer voorkomen.